aries-site/src/pages/components/textinput.mdx

```diff
@@ -117,3 +117,7 @@ Apply `readOnlyCopy` to TextInput to display a copy button in the input.
 >
   <TextInputReadOnlyExample />
 </Example>
+
```

Comment:
Can we move the existing accessibility section down to the bottom of the page and include WCAG compliance as a sub-header under 'Accessibility'

Reply:
Done.